app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::V1::ReservationsController < ApplicationController
+  # before_action :authenticate_user!, only: %i[create index destroy]
```

Comment:
- [ ] Kindly, remove this commented code in order to improve the repo's clenliness.